sdk/apimanagement/api-management-custom-widgets-scaffolder/bin/execute.cjs

```diff
@@ -1,3 +1,4 @@
+#!/usr/bin/env node
```

Comment:
Is this change needed? Same Q for the cjs.map file below

Reply:
Not related to this PR but since I was touching the package.json and rebuild the package the two changes were the result of build.

Reply:
I opened a separate PR for this https://github.com/Azure/azure-sdk-for-js/pull/31732